incubator/hnc/Makefile

```diff
@@ -27,6 +27,8 @@ else
 endif
 endif
 
+# Override this to release from a different repo
+HNC_REPO_USER ?= "kubernetes-sigs"
```

Comment:
nit: This is generally called the org/organization in docker world, not user.

Reply:
Good call. This is for a Github repo and apparently they're called [owners](https://github.com/gitbucket/gitbucket/wiki/List-of-URL-Formats#repositories) there, so I'll make that change. But I already have another dependent PR so I'll make it in the followup change rather than try to merge it in.

/hold cancel